src/main/java/seedu/address/logic/commands/schedule/EditScheduleCommand.java

```diff
@@ -22,55 +24,71 @@
 /**
  * Edits the details of an existing Client in the address book.
  */
-public class RescheduleCommand extends Command {
+public class EditScheduleCommand extends Command {
 
-    public static final String COMMAND_WORD = "reschedule";
+    public static final String COMMAND_WORD = "editschedule";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Reschedule a client with another session. "
-            + "Parameters: INDEX (must be a positive integer)\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": schedules a client with another session. \n"
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: "
+            + PREFIX_CLIENT_INDEX + "CLIENT "
             + PREFIX_SESSION_INDEX + "SESSION "
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_SESSION_INDEX + "1 ";
+            + PREFIX_UPDATED_SESSION_INDEX + "UPDATED SESSION "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_CLIENT_INDEX + "1 "
+            + PREFIX_SESSION_INDEX + "1 "
+            + PREFIX_UPDATED_SESSION_INDEX + "1 ";
 
-    public static final String MESSAGE_EDIT_SCHEDULE_SUCCESS = "Rescheduled : \n%1$s";
+    public static final String MESSAGE_EDIT_SCHEDULE_SUCCESS = "Editschedule : \n%1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_SCHEDULE = "This Schedule overlaps with an existing Schedule";
 
-    private final Index scheduleIndex;
     private final Index sessionIndex;
-    private final RescheduleDescriptor editRescheduleDescriptor;
+    private final Index clientIndex;
+    private final Index updatedSessionIndex;
```

Comment:
should this be encapsulated in editSchedDescriptor?

Reply:
I would think that its good clearer to have the values here too, esp when I'm debugging the tests and its also required in createEditedSchedule method